flocker/control/httpapi.py

```diff
@@ -280,7 +306,8 @@ def saved(ignored):
         outputSchema={
             '$ref':
             '/v1/endpoints.json#/definitions/configuration_datasets'},
-        schema_store=SCHEMAS
+        schema_store=SCHEMAS,
+        get_etag=get_configuration_etag,
```

Comment:
Using the etag from `/configuration/datasets` on `/configuration/datasets/<dataset_id>` isn't supported by the RFC (but I guess is probably a compatible extension to it?)